control-plane/tests/test_service_persistence.py

```
@@ -7548,6 +7548,233 @@ def test_service_creates_approval_bound_action_request_from_reviewed_recommendat
         )
         self.assertEqual(store.list(ActionExecutionRecord), ())
 
+    def test_service_executes_phase20_first_live_action_end_to_end_from_reviewed_recommendation(
+        self,
+    ) -> None:
+        store, service, promoted_case, evidence_id, reviewed_at = (
```

Ruff (0.15.9)

[warning] Unpacked variable `store` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)